package.json

```diff
@@ -21,30 +21,21 @@
   },
   "dependencies": {
     "@cosmostation/cosmosjs": "^0.5.2",
-    "@lunie/cosmos-js": "0.0.22",
-    "@lunie/cosmos-keys": "file:../cosmos-keys",
     "@mesg/api": "^0.2.0",
     "@nuxtjs/google-analytics": "^2.2.3",
     "@nuxtjs/pwa": "^3.0.0-0",
-    "@tendermint/amino-js": "^0.6.2",
-    "bech32": "^1.1.3",
-    "bip32": "^2.0.5",
-    "bip39": "^3.0.2",
     "body-parser": "^1.19.0",
     "cors": "^2.8.5",
-    "crypto-js": "^3.1.9-1",
     "cytoscape": "^3.12.1",
     "cytoscape-dagre": "^2.2.2",
-    "fibers": "^4.0.2",
     "ipfs-http-client": "^40.0.1",
     "markdown-it": "^10.0.0",
     "node-fetch": "^2.6.0",
     "nuxt": "^2.10.2",
     "nuxt-env": "^0.1.0",
     "pluralize": "^8.0.0",
-    "secp256k1": "^4.0.0",
     "tar": "^5.0.5",
-    "tendermint": "^4.0.8",
+    "tendermint": "git+https://github.com/mesg-foundation/js-tendermint.git#beta",
```

Comment:
i got docker image build error
```
npm ERR! syscall spawn git
```
i think git is not install in the docker image

Reply:
removing `alpine` from the docker image did the trick.

Reply:
removing `alpine` from the docker image did the trick.